cyber/doxy-docs/build_doxy_sphinx.sh

```diff
@@ -0,0 +1,4 @@
+rm -rf xml/*
+doxygen Doxyfile
+rm -rf build/* 
+sphinx-build -b html -d build/doctrees   source build/html
```

Comment:
1. Fix the redundant whitespaces.
2. Are these tools available in docker image? If not, plz add it into docker/build/... to better track our tool chain.
3. You may also provide a BUILD file to integrate the building process into the bazel system, as an example:
https://github.com/ApolloAuto/apollo/blob/r3.0.0/modules/perception/cuda_util/BUILD